app/pathogen/arbovirus/analyze/recharts.tsx

```diff
@@ -0,0 +1,906 @@
+"use client";
+
+import { ArboContext } from "@/contexts/arbo-context";
+import { useContext } from "react";
+import {
+  BarChart,
+  Bar,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+  ResponsiveContainer,
+  AreaChart,
+  Area,
+  Label,
+} from "recharts";
+import _ from "lodash";
+import { pathogenColors } from "../dashboard/(map)/MapAndFilters";
+
+//Study by who region and pathogen
+
+// TODO: future enhancement if needed is to extract the function to build the data and keep things cleaner (may not be needed, more readalbe this way
+
+//Study count by pathogen and antibody type
+
+type arbovirusesSF = "DENV" | "ZIKV" | "CHIKV" | "YF" | "WNV" | "MAYV";
+type arboviruses =
+  | "Dengue"
+  | "Zika"
+  | "Chikengunia"
+  | "Yellow Fever"
+  | "West Nile"
+  | "Mayaro";
```

Comment:
Might be worth coordinating with Elias + Gurick for a bit of consistency here. Ex. they have "Mayaro Virus" and "Zika Virus" whereas you haven't put virus here. Pretty small tbh just my consistency itch mostly hahaha

Reply:
Also I think "Chikungunya" might be the correct spelling but I'm open to being wrong here.

Reply:
+1